slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/interchange/InterchangeManager.java

```diff
@@ -13,9 +13,12 @@
 package tech.pegasys.web3signer.slashingprotection.interchange;
 
 import java.io.IOException;
+import java.io.InputStream;
 import java.io.OutputStream;
 
 public interface InterchangeManager {
```

Comment:
Don't this interface makes sense anymore. The export is in the InterchangeV5Manager but the import is just delegated to the InterchangeV5Importer class. Don't think the interface holds any benefit anymore.

Reply:
possibly true - will refactor export into its own class, then see what happens after that.